integration/ethereummock/eth_client.go

```diff
@@ -0,0 +1,43 @@
+package ethereummock
```

Comment:
I think it would be better if we'd had "ethereummock.Node" directly implement "ethclient.Client", instead of going through this wrapper that doesn't really add anything

Reply:
I think it might put more stuff than needed into the Node but we can decouple that further down the line if needed 👍